extensions/wikia/Search/classes/QueryService/Select/AbstractSelect.php

```diff
@@ -347,87 +385,96 @@ protected function sendSearchRequestToClient() {
 		} catch ( \Exception $e ) {
 			if ( $this->config->getError() !== null ) {
 				$this->config->setError( $e );
+
 				return new \Solarium_Result_Select_Empty();
 			} else {
-				$this->config->setSkipBoostFunctions( true )
-				             ->setError( $e );
+				$this->config->setSkipBoostFunctions( true )->setError( $e );
 
 				return $this->sendSearchRequestToClient();
 			}
 		}
 	}
-	
+
 	/**
 	 * This is a hook for child classes to optionally extend
+	 *
 	 * @return AbstractSelect
 	 */
 	protected function prepareRequest() {
 		$config = $this->getConfig();
 		if ( $config->getPage() > 1 ) {
-			$start = ( ( $config->getPage() - 1  ) * $config->getLimit() ) -  $config->mustAddMatchedRecords();
+			$start = ( ( $config->getPage() - 1 ) * $config->getLimit() ) - $config->mustAddMatchedRecords();
 			$config->setStart( $start );
 		}
+
 		return $this;
 	}
-	
+
 	/**
 	 * Allows us to re-search for a collated spellcheck
+	 *
 	 * @param Solarium_Result_Select $result
+	 *
 	 * @return Solarium_Result_Select|\Solarium_Result_Select_Empty
 	 */
 	protected function spellcheckResult( Solarium_Result_Select $result ) {
 		// re-search for spellchecked phrase in the absence of results
 		$service = $this->getService();
 		$config = $this->getConfig();
-		if ( $service->getGlobal( 'WikiaSearchSpellcheckActivated' ) 
-				&& $result->getNumFound() == 0
-				&& !$config->hasMatch() ) {
+		if ( $service->getGlobal( 'WikiaSearchSpellcheckActivated' ) &&
+			$result->getNumFound() == 0 &&
+			!$config->hasMatch()
+		) {
 			if ( $collation = $result->getSpellcheck()->getCollation() ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)